variables.tf

```
@@ -178,4 +196,16 @@ variable "vpn_connection_tunnel2_startup_action" {
   type        = string
   description = "The action to take when the establishing the tunnel for the second VPN connection. By default, your customer gateway device must initiate the IKE negotiation and bring up the tunnel. Specify start for AWS to initiate the IKE negotiation. Valid values are add | start."
   default     = "add"
-}
\ No newline at end of file
+}
+
+variable "vpn_connection_tunnel2_cloudwatch_log_enabled" {
+  type        = bool
+  description = "Enable or disable VPN tunnel logging feature for the tunnel"
+  default     = false
+}
+
+variable "vpn_connection_tunnel2_cloudwatch_log_output_format" {
+  type        = string
+  description = "Set log format for the tunnel. Default format is json. Possible values are: json and text"
+  default     = null
```

Should this have a default of `"json"` so that it is more clear? Not sure... If you've used this configuration and this works as it expected (which it should) then no changes needed here, but just calling this out that it's weird that we're saying "Default is JSON, but `default` is `null`". Feels more explicit to specify JSON as the default.

---

You have a good point. :) 

```suggestion
  default     = "json"
```